components/accessibility-table.js

```diff
@@ -0,0 +1,97 @@
+// node_modules
+import PropTypes from "prop-types";
+import { MagnifyingGlassIcon } from "@heroicons/react/20/solid";
+// components
+import { DataGridContainer } from "./data-grid";
+import SortableGrid from "./sortable-grid";
+import { useState } from "react";
+import { sanitizedString } from "../lib/general";
+
+const accessibilityDataColumns = [
+  {
+    id: "method",
+    title: "Method",
+  },
+  {
+    id: "peak",
+    title: "Peak",
+    display: ({ source }) => `${source.chrom}:${source.start}-${source.end}`,
+    sorter: (source) => Number(source.start),
+  },
+  {
+    id: "biosample_ontology",
+    title: "Biosample",
+    display: ({ source }) =>
+      source.biosample_ontology ? source.biosample_ontology.term_name : "",
+  },
+  {
+    id: "organ_slims",
+    title: "Organ",
+    display: ({ source }) =>
+      source.biosample_ontology?.organ_slims?.length > 0
+        ? source.biosample_ontology.organ_slims.join(", ")
+        : "",
+  },
+  {
+    id: "dataset",
+    title: "Dataset",
+    display: ({ source }) => (
+      <a href={source.dataset}>{source.dataset.split("/")[4]}</a>
+    ),
+  },
+  {
+    id: "file",
+    title: "File",
+    display: ({ source }) => (
+      <a href={`https://encodeproject.org/files/${source.file}/`}>
+        {source.file}
+      </a>
+    ),
+  },
+  {
+    id: "value",
+    title: "Value",
+  },
+];
+
+/**
+ * Display a sortable table of the given accessibility data.
+ * The data can be filtered by biosample term name.
+ */
+export default function AccessibilityDataTable({ data }) {
+  const [textInput, setTextInput] = useState("");
+  data =
+    textInput === ""
+      ? data
+      : data.filter((dataset) =>
+          sanitizedString(dataset.biosample_ontology.term_name).includes(
+            sanitizedString(textInput)
+          )
+        );
+  return (
+    <div className="grid gap-y-2">
+      <label
+        htmlFor="email"
+        className="relative text-gray-400 focus-within:text-gray-600 block"
+      >
```

Comment:
The `<input>` needs a `name` property corresponding to the `htmlFor` (`for`) on the label so that screen readers know what label to say for the input. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label#try_it

Reply:
Thanks Forrest. Learned something new again. I found the style I needed online and just copied it over without checking into the detail. Actually since my input is nested in the label the `for` and `id` attributes are not needed because the association is implicit. I removed the `htmlFor` and it works!